packages/lambda-tiler/src/wmts.capability.ts

```diff
@@ -19,6 +19,15 @@ function wgs84Extent(layer: TileSet): BBox {
     return Projection.get(layer.tileMatrix).boundsToWgs84BoundingBox(layer.extent);
 }
 
+/**
+ * Default the tile matrix id to the projection of the TileMatrixSet
+ */
+function getTileMatrixId(tileMatrix: TileMatrixSet): string {
```

Comment:
I am not a huge fan of this, we should maybe change all WMTS's to use the tile matrix id but this would be a user facing change, 

should validate with @JBLINZ if we are ok changing the name and make a specific pull request for it.

Reply:
Agree, this seems just a temporary solution.

Reply:
@JBLINZ has agreed to move to the Tile Matrix Id as the layer names I will make another pull request for it.